build.githubactions.ps1

```diff
@@ -55,19 +55,25 @@ param(
     $NuspecFilePath,
 
     [string]
-    $RelativeRepoPath
-)
-
-$ErrorActionPreference = 'Stop'
+    $RelativeRepoPath,
 
-$error.Clear()
+    [string]
+    $StandardVersion,
 
-& "$PSScriptRoot/../Ed-Fi-ODS-Implementation/logistics/scripts/modules/load-path-resolver.ps1"
+    [string]
+    $ExtensionVersion
+)
 
 $newRevision = ([int]$BuildCounter) + ([int]$BuildIncrementer)
 $version = "$InformationalVersion.$newRevision"
 $packageOutput = "$PSScriptRoot/NugetPackages"
-$packagePath = "$packageOutput/$PackageName.$version.nupkg"
+
+if ($PackageName.Contains(".Extensions.")) { 
```

Comment:
this can be simplified if we pass the package name with the version and standard included

Reply:
Fixed it . Thanks